### PR TITLE
Introduced dashboard stats for nerds

### PR DESCRIPTION
**Pull request by lotas**
Introduces stats for logged in users:
image

### CODE REVIEW

**Comment by lotas** [comment]
> This is really cool!
> 
> A `?` or some icon might be better than `0` for the error condition?

Yeah, I was thinking if only using different color would be enough, or showing something else. Will try with different icons

**Comment by Eijebong** [comment]
I would personally hide the cards if you can't fetch info? Thinking about a locked down instance where everything would be missing, it would look a bit weird

**Comment by lotas** [comment]
> I would personally hide the cards if you can't fetch info? Thinking about a locked down instance where everything would be missing, it would look a bit weird

@Eijebong yeah.. so the trick would be to know where you have access to. We'll need to do an extra call to expand all scopes for the current user to know where we have access to.
But I understand your concern, if your users do not have permissions, it doesn't make much sense to show it. 
And we also don't want to make UI add and remove those cards randomly.

I think we can initially hide everything, put some spinner or "loading.." and wait until requests are finished to see what cards we need to show. So user will always only see what he can see. 

Another option could be to put this feature behind a configuration option `SITE_SPECIFIC` that can disable this feature for such deployments that do not want to show any of that info

**Comment by Eijebong** [comment]
Yeah you could start with no cards displayed and display them as you get data. Not sure if the current implementation is synchronous or not but it might also help with loading time.

File: ui/src/components/StatusDashboard/summarizeAuthorization.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  return [
+    {
+      title: 'Total Clients / Active clients',
+      hint: `Clients that were active in the last ${ACTIVE_DAYS} days`,
+      value: `${format(clientsCount)} / ${clientsRecent}`,
+      link: '/auth/clients',
+      error: clients.error?.message,
+      loading: clients.loading,
+    },
```
[the remark this answers is not in the document]
Comment (matt-boris) [reply ui/src/components/StatusDashboard/summarizeAuthorization.js:43]: Was there a reason you didn't have `format()` for `clientsRecent` here? Just to call out that I added it in, in my suggestion.

[the remark this answers is not in the document]
Comment (lotas) [reply ui/src/components/StatusDashboard/summarizeAuthorization.js:43]: thanks! probably missed :)

File: ui/src/components/StatusDashboard/summarizeWorkerPools.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    {
+      title: 'Running Capacity',
+      value: format(runningCapacity),
+      link,
+      error: workerPools.error?.message,
+      loading: workerPools.loading,
+    },
+    {
+      title: 'Requested Capacity',
+      value: `format(requestedCapacity),
```
[the remark this answers is not in the document]
Comment (lotas) [reply ui/src/components/StatusDashboard/summarizeWorkerPools.js:92]: too late, thanks :D fixed locally

**Review by matt-boris — APPROVED** [review]
Nice! 🎉

**Pull request merged**